docs/samples/specification/directives/generated/azure/directives/sample/aio/operations/__init__.py

```diff
@@ -8,6 +8,11 @@
 
 from ._polling_paging_example_operations import PollingPagingExampleOperationsMixin
 
+from ._patch import __all__ as _patch_all
+from ._patch import *  # pylint: disable=unused-wildcard-import
+from ._patch import patch_sdk as _patch_sdk
 __all__ = [
     'PollingPagingExampleOperationsMixin',
 ]
+__all__.extend([p for p in _patch_all if p not in __all__])
+_patch_sdk()
```

Comment:
It seems lack annotation like: https://github.com/Azure/autorest.python/pull/1153/files#r823408279

Reply:
see comment below